reana_cluster/backends/kubernetes/templates/deployments/workers-template.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
       - name: {{worker.WORKFLOW_ENGINE.NAME}}-{{worker.EXPERIMENT}}-worker
-        image: {{WORKFLOW_ENGINE_IMAGE}}
+        image: {{WORKFLOW_ENGINE_YADAGE_IMAGE}}
```

Comment:
Regarding https://github.com/reanahub/reana-workflow-engine-yadage/pull/43#discussion_r160975502 discussion I think this environment variable should be implementor agnostic name wise because it will be present in any workflow engine, it is imposed by our "framework" to plug an engine to the current architecture.